src/Controller/Rest/ApiController.php

```diff
@@ -99,4 +111,13 @@ public function generateApi(Api $api, SymfonyGenerator $symfonyGenerator): Respo
 
         return $this->getApi($api);
     }
+
+    /**
+     * @param Request $request
+     * @return \App\Entity\User|mixed|null
+     */
+    private function getUserFromRequest(Request $request)
+    {
+        return $this->userService->getUser($request->get('userId'));
+    }
```

Comment:
Ideally move this method to the RestController and have a better way to handle the logged in user, maybe in the controller?